### PR TITLE
Add raincloud plots for one-sample t-tests and paired differences

### PR DESCRIPTION
**Pull request by maltelueken**
Adds the raincloud plots for one-sample t-tests and for the differences between measurement points in paired t-tests. Both can also be plotted horizontally.  The difference plot does not have an y-axis label, does someone maybe have a suggestion what this could be?

Difference plot:
raincloud_plot_difference

One-sample t-test:
raincloud_plot_onesample

### CODE REVIEW

File: R/commonTTest.R [hunk 1]
```diff
@@ -125,13 +125,13 @@
 }
 
 .ttestDescriptivesContainer <- function(jaspResults, options) {
-  if(!options$descriptives && !options$descriptivesPlots && !options$descriptivesPlotsRainCloud) return()
+  if(!options$descriptives && !options$descriptivesPlots && !options$descriptivesPlotsRainCloud && !options$descriptivesPlotsRainCloudDifference)
```
Comment (Kucharssim) [inline R/commonTTest.R:128]: This gives me the following error in the classical one-sample t-test:
Screenshot 2020-12-04 at 21 45 53

Perhaps you want to use `isFALSE()` here

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks quite good! The one sample gives an error, see the comment.

> The difference plot does not have an y-axis label, does someone maybe have a suggestion what this could be?

Why not just "variableName1 - variableName2"?

File: R/commonTTest.R (hunk 1 above)
Comment (maltelueken) [reply R/commonTTest.R]: Yes, that solves the issue! I don't know why it didn't give me an error with the unit tests before...

**Comment by maltelueken** [comment]
I added the y-label "Measurement 1 - Measurement 2" as well!

**Review by Kucharssim — APPROVED** [review]
Cool, it's fine now!

**Pull request merged**